modules/oe_content_organisation/oe_content_organisation.info.yml

```diff
@@ -14,6 +14,8 @@ dependencies:
   - inline_entity_form:inline_entity_form
   - oe_media:oe_media_avportal
   - description_list_field:description_list_field
+  - oe_media:oe_media
+  - oe_content:oe_content_organisation_person
```

Comment:
We need to remove this dependency, the whole point is that users can enable the submodule if they want.